docs/config.rst

```diff
@@ -161,7 +161,7 @@ Default is to disallow fetching of icons from remote sources.
 --------------
 Allows the rendering of inline marker icons or base64 urls.
 For security reasons only allow this if you can control the origins from where the markers are fetched!
-Not used by default.
+Not used by default. 
```

Comment:
what a weird lint change

Reply:
My bad, I think I used the linter wrong. Should be good now though